env/requirements.yaml

```diff
@@ -33,8 +33,8 @@ dependencies:
   repository: https://charts.gitlab.io/
   version: 0.1.12
 - name: jx-app-ui
-  repository: http://chartmuseum.jenkins-x.io
-  version: 0.1.211
+  repository: https://chartmuseum-jx.jenkins-x.live
+  version: 0.1.213
 - name: jx-app-ui
```

Comment:
oh, for some reason this is redundant. @cagiti can you remove it in your PR as well? (otherwise it will fail because either due to the duplication or because the next entry overrides this one)